### ISSUE TITLE
Multi-server support

### ISSUE THREAD

**Issue by Windamier** [Luzefiru/wuwatracker#67]
# Problem

For those who might play on several of the regional servers (Asia, SEA, America, etc), it would be lovely if the pull histories for all of them could be attached to the same google drive account.

## Possible Solution

In cases such as StarRailStation, Genshin-center or Paimon.moe, users are able to create separate profiles to keep track of multiple servers. Even just a little drop-down menu to toggle between regions. Unfortunately I'm not familiar enough with coding to offer a proper solution.

## Additional Context or Screenshots

![image](https://github.com/Luzefiru/wuwatracker/assets/173452619/cdcae425-3bfe-4544-a303-082fe611f25a)
![image](https://github.com/Luzefiru/wuwatracker/assets/173452619/28b68a66-4f2e-4aef-8be8-0a9476c1f60e)

**Comment by Luzefiru** [comment]
> it would be lovely if the pull histories for all of them could be attached to the same google drive account

Agreed. I'll be revisiting our saving feature in the future and I'll take this into careful consideration.

Thanks for suggesting this! 👍

**Comment by My1** [comment]
stupid idea, if the pull data you import contains a UID you could even use that as an automatic filter

**Comment by Luzefiru** [comment]
We have profiles now!
